packages/application-shell/src/index.ts

```diff
@@ -37,6 +39,12 @@ export {
   entryPointUriPathToResourceAccesses,
 } from '@commercetools-frontend/application-config/ssr';
 
+// This is for internal purposes only.
+// Please do not rely on it as it might change at any time.
+export const _graphqlQueries = {
+  FetchCustomViewsQuery,
```

Comment:
Exporting this query so we don't need to duplicate it in the visual testing application.